web/components/Footer.module.scss

```diff
@@ -3,55 +3,50 @@
 .container {
     display: flex;
     position: absolute;
-    width: 100%;
     height: 18vh;
     background-color: $background;
     align-items: center;
-    justify-content: center;
+    // justify-content: center;
```

Comment:
is there a reason that you are keeping this comment?

Reply:
not any good ones removing that now